Basic_C_Examples/basic.c

```diff
@@ -39,11 +39,24 @@ int main()
 
 	printf("Integer is %d\n", a);			//print integer value
 	printf("Char is %c\n\n", b);			//print character value
+	
+	
+	
+	/*Takes the value entered by the user and displays the output*/
+        int i;
+        char h[20];
+        printf("Enter your name: \n"); 
+        scanf("%s",h);             //Takes input from the user
+        printf("Your name is: %s\n",h);
+        printf("Enter a number: \n");
+        scanf("%d",&i);

```

Comment:
If the input number does not fit in an signed integer it will roll over, if it is not a number it will set i to 0. Some error handling for these cases are good practice, see https://wiki.sei.cmu.edu/confluence/display/c/INT05-C.+Do+not+use+input+functions+to+convert+character+data+if+they+cannot+handle+all+possible+inputs